container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -328,10 +329,28 @@ function run_git_tempest {
     tempest init openshift
     pushd $TEMPEST_DIR
 
-    discover_tempest_config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES} \
+    # We're running cleanup only under certain circumstances
+    if [[ ${TEMPEST_DEBUG_MODE} == false ]] && [[ ${TEMPEST_CLEANUP} == true ]]; then
```

Comment:
@kopecmartin, do we really want the cleanup to be cancelled if `debug` mode is turned on? What if somebody wants to run in debug mode and keep `cleanup` turned on. 

If we do this we can leave the decision up to the user.
```
if [[ ${TEMPEST_CLEANUP} == true ]]; then
```